systemtest/src/test/java/io/strimzi/systemtest/bridge/HttpBridgeST.java

```diff
@@ -124,23 +119,19 @@ void testReceiveSimpleMessage() {
             .withPort(TestConstants.HTTP_BRIDGE_DEFAULT_PORT)
             .withDelayMs(1000)
             .withPollInterval(1000)
-            .withNamespaceName(Environment.TEST_SUITE_NAMESPACE)
+            .withNamespaceName(testStorage.getNamespaceName())
             .build();
 
         resourceManager.createResourceWithWait(kafkaBridgeClientJob.consumerStrimziBridge());
 
         // Send messages to Kafka
-        KafkaClients kafkaClients = new KafkaClientsBuilder()
-            .withTopicName(testStorage.getTopicName())
-            .withMessageCount(testStorage.getMessageCount())
+        KafkaClients kafkaClients = ClientUtils.getInstantPlainClientBuilder(testStorage)
```

Comment:
your idea is providing argument on "every place" `myClusterName`, or `KafkaResources.plainBootstrapAddress(myClusterName)` ? 


getInstantPlainClientBuilder(testStorage, **myClusterName**) would not be usable in places where we need different port, (i know it is not so common but it can be misleading) 

or 
getInstantPlainClientBuilder(testStorage, **KafkaResources.plainBootstrapAddress(myClusterName)**)
which i did not used originally because it seemed a bit to wordy to me. 
and usage of explicit .`withBootstrapAddress` did not seem as that big overkill. 

With that said if you can clarify which option you see as most feasible I am glad to proceed with any of them.

Reply:
no I mean another encapsulation
so by default it will take `KafkaResources.plainBootstrapAddress(myClusterName)`, but you will be able to specify the bootstrapServer

so something like this:

```
public KafkaClients getInstantPlainClientBuilder(TestStorage testStorage) {
    return getInstantPlainClientBuilder(testStorage, KafkaResources.plainBootstrapAddress(testStorage.getClusterName());
}

public KafkaClients getInstantPlainClientBuilder(TestStorage testStorage, String bootstrapServer) {
.... (magic as you have today, just with bootstrapServer set from the parameter and not from the KafkaResources method
}
```